draft-ietf-acme-acme.md

```diff
@@ -1950,16 +1950,16 @@ client then provisions the key authorization as a resource on the HTTP server
 for the domain in question.
 
 The path at which the resource is provisioned is comprised of the fixed prefix
-".well-known/acme-challenge/", followed by the "token" value in the challenge.
-The value of the resource MUST be the ASCII representation of the key
-authorization.
+".well-known/acme-challenge/", followed by the SHA-256 digest of the "token"
+value in the challenge. The value of the resource MUST be the ASCII
```

Comment:
You need to specify how this is encoded.  I would be inclined to use the same base64url encoding we use elsewhere, if only to avoid ambiguity about upper case / lower case hex.

Why not just digest the entire key authorization?  Then you're basically just requiring the client to give you a hash pre-image.